pages/cart.vue

```diff
@@ -80,152 +73,82 @@
               description="Looks like you haven’t added any items to the cart yet. Start
                 shopping to fill it in."
             />
-            <SfButton
-              class="sf-button--full-width color-primary empty-cart__button"
-              >Start shopping</SfButton
-            >
+            <SfButton class="sf-button--full-width color-primary empty-cart__button">{{
+              $t("Start shopping")
+            }}</SfButton>
           </div>
         </transition>
       </div>
     </div>
   </div>
 </template>
 <script>
-import {
-  SfCollectedProduct,
-  SfButton,
-  SfImage,
-  SfProperty,
-  SfHeading,
-  SfBreadcrumbs,
-  SfOrderSummary,
-} from "@storefront-ui/vue";
-export default {
+import { SfButton, SfImage, SfHeading, SfBreadcrumbs, SfInput } from "@storefront-ui/vue"
+import { useAsync } from "@nuxtjs/composition-api"
+import { defineComponent } from "@vue/composition-api"
+import { usePurchaseLocation, useCart } from "@/composables"
+import useUiState from "@/composables/useUiState"
+import KiboCollectedProduct from "@/components/KiboCollectedProduct.vue"
+import { cartGetters, productGetters, storeLocationGetters } from "@/composables/getters"
+
+export default defineComponent({
   name: "DetailedCart",
   components: {
-    SfCollectedProduct,
     SfBreadcrumbs,
     SfImage,
     SfButton,
     SfHeading,
-    SfProperty,
-    SfOrderSummary,
+    SfInput,
+    KiboCollectedProduct,
   },
-  data() {
+  setup() {
+    const { toggleStoreLocatorModal } = useUiState()
+    const { purchaseLocation, load: loadPurchaseLocation } = usePurchaseLocation()
+    const { cart, load: loadCart } = useCart()
+
+    const breadcrumbs = [
+      {
+        text: "Home",
+        link: "/",
+      },
+      {
+        text: "Cart",
+        link: "/cart",
+      },
+    ]
+
+    useAsync(async () => {
+      await loadPurchaseLocation()
```

Comment:
We already trigger loadPurchaseLocation in the app header.
const { purchaseLocation } = usePurchaseLocation() should return the same object reference, without the extra http call

Reply:
Resolved